src/main/java/taxi/controller/authentification/LoginController.java

```diff
@@ -0,0 +1,46 @@
+package taxi.controller.authentification;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import taxi.exception.AuthenticationException;
+import taxi.lib.Injector;
+import taxi.model.Driver;
+import taxi.service.AuthenticationService;
+
+@WebServlet("/login")
+public class LoginController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("taxi");
+    private final AuthenticationService authenticationService;
+
+    public LoginController() {
+        authenticationService = (AuthenticationService)
+                injector.getInstance(AuthenticationService.class);
+    }
```

Comment:
Do we need here do this with constructor? Example, in GetMyCurrentCarsController you initialize variable already when declaring it